### ISSUE TITLE
Better way to handle FormData from browser?

### ISSUE THREAD

**Issue by msheakoski** [fastify/fastify-multipart#61]
I have read through the README and code, and it seems like the only way to use this plugin currently is with the expectation of a file being uploaded.

In my case, I am generating FormData in the browser and posting it via fetch, so it seems strange that I need to define a file handler just to get an instance of busboy?
```javascript
const mp = req.multipart(handler, function (err) {
  console.log('upload completed')
  reply.code(200).send()
})
```

I am only interested in the "field" and "finish" events to read FormData, so being forced to write the code above for file handling does not make sense:
```javascript
const formData = new Map();

mp.on('field', function (key, value) {
  formData.set(key, value);
});

mp.on("finish", function() {
  // do stuff with formData
});
```

I think that the current API and plugin is not very clear that it is mainly useful for file uploads.

**Comment by msheakoski** [comment]
Thank you for the update, @Eomm! For anyone that might come across this issue, here is how I currently solved it:
```javascript
server.post("/api/example", async (req, reply) => {
    const formData = {};
    const mp = req.multipart(() => undefined, onFinish);
    mp.on("field", (name, value) => formData[name] = value);
    async function onFinish(error) {
        if (error) {
            // handle error here
        }        
        console.log(formData.lastName); // do stuff with formData here
        reply.send();
    }
});
```

**Comment by msheakoski** [comment]
I have only used fastify and busboy for 1 day, so I may not have a full understanding of the internals, but here is my idea of a possible API:

```javascript
server.post("/api/example", async (req, reply) => {
    const formData = {};

    try {
        // internally, busboy/mp.on("finish") would resolve this promise
        await req.multipart((mp) => {
            mp.on("field", (name, value) => {
                formData[name] = value
            });
            mp.on("file", (fieldname, file, filename, encoding, mimetype) => {
                // save file to disk
            });
        });
    } catch (error) {
        // handle error, return 500 response
    }

    // do stuff with formData
    console.log(formData.lastName);
    return reply.send();
});
```

**Comment by Eomm** [comment]
Released the `addToBody` parameter in [v0.8.0](https://github.com/fastify/fastify-multipart/releases/tag/v0.8.0) 👍
